src/SFA.DAS.Commitments.Database/Tables/ClientOutboxData.sql

```diff
@@ -5,12 +5,13 @@
     [CreatedAt] DATETIME NOT NULL,
     [Dispatched] BIT NOT NULL DEFAULT(0),
     [DispatchedAt] DATETIME NULL,
-    [Operations] NVARCHAR(MAX) NOT NULL
+    [Operations] NVARCHAR(MAX) NOT NULL,
+    [PersistenceVersion] VARCHAR(23) NOT NULL DEFAULT '1.0.0'
 )
 GO
 
-CREATE INDEX [IX_CreatedAt] ON [dbo].[ClientOutboxData] ([CreatedAt] ASC) WHERE [Dispatched] = 0
+CREATE INDEX [IX_CreatedAt_PersistenceVersion] ON [dbo].[ClientOutboxData] ([CreatedAt] ASC, [PersistenceVersion] ASC) WHERE [Dispatched] = 0
```

Comment:
I can't remember why this needs a where clause. Would adding the Dispatched field to the index not create the same logical outcome? The second index would not be needed?